arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -0,0 +1,781 @@
+        "description": "Gets all the management locks of a subscription.",
+        "parameters": [
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ManagementLockListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/ManagementLockObject"
+      }
+    }
+  },
+  "definitions": {
+    "DeploymentExtendedFilter": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets the provisioning state."
+        }
+      },
+      "description": "Deployment filter."
+    },
+    "GenericResourceFilter": {
+      "properties": {
+        "resourceType": {
+          "type": "string",
+          "description": "Gets or sets the resource type."
+        },
+        "tagname": {
+          "type": "string",
+          "description": "Gets or sets the tag name."
+        },
+        "tagvalue": {
+          "type": "string",
+          "description": "Gets or sets the tag value."
+        }
+      },
+      "description": "Resource filter."
+    },
+    "ResourceGroupFilter": {
+      "properties": {
+        "tagName": {
+          "type": "string",
+          "description": "Gets or sets the tag name."
+        },
+        "tagValue": {
+          "type": "string",
+          "description": "Gets or sets the tag value."
+        }
+      },
+      "description": "Resource group filter."
+    },
+    "ManagementLockOwner": {
+      "properties": {
+        "applicationId": {
+          "type": "string",
+          "description": "Gets or sets the application Id of the management lock owner."
+        }
+      }
+    },
+    "ManagementLockProperties": {
+      "properties": {
+        "level": {
+          "type": "string",
+          "description": "Gets or sets the lock level of the management lock.",
+          "enum": [
+            "NotSpecified",
+            "CanNotDelete",
+            "ReadOnly"
+          ],
+          "x-ms-enum": {
+            "name": "LockLevel",
+            "modelAsString": true
+          }
+        },
+        "notes": {
+          "type": "string",
+          "description": "Gets or sets the notes of the management lock."
+        },
+        "owners": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementLockOwner"
+          },
+          "description": "Gets or sets the owners of the management lock."
```

Comment:
Can we please remove the "Gets or sets" throughout this file? If you use the latest autorest.exe to gen the sdk, this will result in duplicate gets or sets. The reason is, gets or sets is C# specific, but we use this file to generate SDK for other languages as well, where gets or sets doesn't apply. I did this for our subscription file [here](https://github.com/Azure/azure-rest-api-specs/pull/503)